src/module/counter.js

```diff
@@ -0,0 +1,6 @@
+const counterItem = (arr) => arr.length;
+
+const countercomment = (comm) => comm.length;
```

Comment:
[OPTIONAL] JavaScript function names are also case-sensitive. So, similar to variables, the camelCase approach is the recommended way to declare function names. In addition to that, you should use descriptive nouns and verbs as prefixes. if we declare a function to count comments, the function name should be `countComments`.